src/pages/patterns/drag-and-drop.mdx

```diff
@@ -0,0 +1,596 @@
+the dropzone will appear after the user drags the item so only “drop” is mentioned in the content.
+Include the consequence if it’s important for the user to understand what will happen when they drop
+the item in this dropzone. Content, together with supporting icons and colors will make the user
+confident in the end result.
+
+![Three different examples showing a default dropzone, a dropzone with a danger theme applied,
+and a highlighted dropzone.](figma:patterns-dnd-formatting-dropzone-examples.png)
+
+1. **Default dropzone** without icon
+2. **Danger dropzone** with a theme and icon applied
+3. **Highlighted dropzone** with a custom icon
+
+Use icons to reinforce the labels when needed. Hide decorative icons from assistive technologies.
+
+#### Empty and populated dropzones
+
+Use borders and labels to make empty dropzones discoverable. Keep populated dropzones simple by showing only items inside.
+Always provide the label for screen readers, even if it's not visible on the screen.
+
+![An empty dropzone on the left and a group with several draggable items.](figma:patterns-dnd-formatting-dropzone-empty-populated.png)
+
+1. Include content that will explain how to interact with the empty dropzone
+2. Populated dropzones don’t need a supporting call-to-action
+
+#### Dropzones as items
+
+Provide visual affordance when items are both draggable and potential dropzones.
+
+![Example of moving a draggable item and replacing another item with it.](figma:patterns-dnd-formatting-dropzone-replace.png)
+
+1. Item with default styling
+2. Item as an active dropzone with a dashed border
+3. Item as a highlighted dropzone
+
+## Flows
+
+### Reordering a list
+
+This is the most basic example of drag-and-drop. Interaction is discoverable through a grip handle on the item.
+See [Draggable tables](https://garden.zendesk.com/components/table#draggable) for reordering rows.
+
+Once the item is grabbed, empty space proportional to the grabbed item will communicate where the item will
+be placed after release. The dragged item should follow the cursor around the screen without any delay.
+
+Once the item is released on the desired location, the movement is instantaneous. If the item is released
+outside of the dropzone container, it snaps back into its original place.
+
+![Four draggable items each containing a description and an overflow icon with one item having a hover state.](figma:patterns-dnd-flows-reorder-1.png)
+![Four draggable items each containing a description and an overflow icon with the second item being dragged.](figma:patterns-dnd-flows-reorder-2.png)
+![Four draggable items each containing a description and an overflow icon with the second item being dragged
+to the third place.](figma:patterns-dnd-flows-reorder-3.png)
+![Four draggable items each containing a description and an overflow icon in the updated order.](figma:patterns-dnd-flows-reorder-4.png)
```

Comment:
The whitespace between these images is competing with the whitespace shown for dropzone. Should some kind of divider (i.e. `---` hr 🤷 ) or more whitespace be added to help differentiate?

Reply:
Very good point. I have increased the image height for the first flow to include more top and bottom padding and also added `---`. It seems much better now.

<img width="1509" alt="image" src="https://user-images.githubusercontent.com/773721/233048984-030ce767-400e-4e53-83a6-966be45f94a0.png">